Gems/LandscapeCanvas/Code/Source/Editor/Core/Core.h

```diff
@@ -44,6 +44,7 @@ namespace LandscapeCanvas
     static const QString GRADIENT_GENERATOR_TITLE = QObject::tr("Gradient");
     static const QString GRADIENT_MODIFIER_TITLE = QObject::tr("Gradient Modifier");
     static const QString VEGETATION_AREA_TITLE = QObject::tr("Vegetation Area");
+    static const QString TERRAIN_TITLE = QObject::tr("Terrain");
```

Comment:
these probably won't translate correctly if the language is changed at any point.

you probably want to do something like this: https://doc.qt.io/qt-6/i18n-source-translation.html#using-qt-tr-noop-and-qt-translate-noop-in-c